src/test/java/jp/ikedam/jenkins/plugins/extensible_choice_parameter/ExtensibleChoiceParameterDefinitionSimpleTest.java

```diff
@@ -206,425 +200,4 @@ public void testExtensibleChoiceParameterDefinition_editable()
-            ExtensibleChoiceParameterDefinition target = new ExtensibleChoiceParameterDefinition(
-                    name,
-                    provider,
-                    false,
-                    description
-            );
-            assertEquals("Non-Editable, in choices, the last", new StringParameterValue(name, defaultChoice, description), target.getDefaultParameterValue());
-        }
-        
-        // Editable, not in choices
-        {
-            String defaultChoice = "value4";
-            ChoiceListProvider provider = new MockChoiceListProvider(Arrays.asList("value1", "value2", "value3"), defaultChoice);
-            ExtensibleChoiceParameterDefinition target = new ExtensibleChoiceParameterDefinition(
-                    name,
-                    provider,
-                    true,
-                    description
-            );
-            assertEquals("Editable, in choices", new StringParameterValue(name, defaultChoice, description), target.getDefaultParameterValue());
-        }
-        
-        // Non-editable, not in choices
-        {
-            String defaultChoice = "value4";
-            ChoiceListProvider provider = new MockChoiceListProvider(Arrays.asList("value1", "value2", "value3"), defaultChoice);
-            ExtensibleChoiceParameterDefinition target = new ExtensibleChoiceParameterDefinition(
-                    name,
-                    provider,
-                    false,
-                    description
-            );
-            try
-            {
-                assertEquals("Non-Editable, not in choices", new StringParameterValue(name, defaultChoice, description), target.getDefaultParameterValue());
-                assertTrue("Not reachable", false);
-            }
-            catch(IllegalArgumentException e)
-            {
-                assertTrue("Non-Editable, not in choices", true);
-            }
-        }
-        
-        // no choice is provided and non-editable. returns null.
-        {
-            ExtensibleChoiceParameterDefinition target = new ExtensibleChoiceParameterDefinition(
-                    name,
-                    new MockChoiceListProvider(new ArrayList<String>(0), null),
-                    false,
-                    description
-            );
-            assertEquals("no choice is provided and non-editable", null, target.getDefaultParameterValue());
-        }
-        
-        // no choice is provided and editable. returns null.
-        {
-            ExtensibleChoiceParameterDefinition target = new ExtensibleChoiceParameterDefinition(
-                    name,
-                    new MockChoiceListProvider(new ArrayList<String>(0), null),
-                    true,
-                    description
-            );
-            assertEquals("no choice is provided and editable", null, target.getDefaultParameterValue());
-        }
-    }
 }

```

Comment:
Above tests are moved to `Above tests are moved from `ExtensibleChoiceParameterDefinitionJenkinsTest` as those tests get to require Jenkins instance and `ExtensibleChoiceParameterDefinitionSimpleTest` doesn't set up Jenkins instance to save test execution time.